docs/using-easybuild.md

```diff
@@ -752,3 +757,19 @@ Dry run: printing build status of easyconfigs and dependencies
 
     Modifying the software version does **not** trickle down the entire software stack, even when combined with `--robot`,
     since a software version is tied to a particular software package.
+
+## Overview of basic module commands  {: #module_cmds_overview }
+
+In case you are not familiar with using modules, here is a simple cheatsheet of most common module commands used in combination with Easybuild:
+
+*(`module avail`)* - list the modules that are currently available to load 
+
+*(`module load foss/2022a`)* - load the module `foss/2022a`
+
+*(`module list`)* - list currently loaded modules
+
+*(`module show foss/2022a`)* - see contents of the module `foss/2022a` (shows the module functions instead of executing them)
+
+*(`module unload foss/2022a`)* - unload the module `foss/2022a`
+
+*(`module purge`)* - unload all currently loaded modules
```

Comment:
```suggestion
*`module avail`* - list the modules that are currently available to load 

*`module load foss/2022a`* - load the module `foss/2022a`

*`module list`* - list currently loaded modules

*`module show foss/2022a`* - see contents of the module `foss/2022a` (shows the module functions instead of executing them)

*`module unload foss/2022a`* - unload the module `foss/2022a`

*`module purge`* - unload all currently loaded modules
```
Drop the brackets but keep the italics, I'm going to merge this rather than hassle you agin